lib/hammer_cli_foreman/commands.rb

```diff
@@ -244,7 +244,7 @@ def self.output(definition = nil, &block)
       super
       begin
         output_definition.update_field_sets('THIN', (searchables.for(resource).map(&:name) + ['id']).map(&:to_sym))
-      rescue StandardError => e
+      rescue StandardError => _
```

Comment:
Isn't this the same?
```suggestion
      rescue StandardError
```